### PR TITLE
f-rating@v0.12.0 - Add isUserRating.

### PR DESCRIPTION
**Pull request by kevinrodrigues**
### Added
- `isUserRating` prop to control when (You) is displayed.
- Tests included both unit and visual.

## screenshots

Screenshot 2022-11-21 at 11 02 37
Screenshot 2022-11-21 at 11 02 49

### CODE REVIEW

File: packages/components/molecules/f-rating/src/components/_tests/Rating.test.js [hunk 1]
```diff
@@ -211,6 +211,72 @@ describe('Rating', () => {
...
+
+                        // Act & Assert
+                        expect(wrapper.vm.shouldDisplayUserOwnRating).toBe(false);
+                    });
+                });
+
+                describe('and `hasRatingAvailable` is falsey and `isUserRating` is truthy', () => {
+                    it('should return false', () => {
+                        // Arrange
+                        propsData.starRating = 2;
```
Comment (xander-marjoram) [inline packages/components/molecules/f-rating/src/components/_tests/Rating.test.js]: I don't think it makes sense to set the star rating if there are no ratings available

File: packages/components/molecules/f-rating/stories/RatingMultiStar.stories.js [hunk 2]
```diff
@@ -25,7 +25,8 @@ export const RatingComponent = (args, { argTypes }) => ({
                     v-bind="$props"
                     :starRatingSize="rating.starRatingSize"
                     :starRating="rating.starRating"
-                    :reviewCount="rating.reviewCount" />
+                    :reviewCount="rating.reviewCount"
```
Comment (xander-marjoram) [inline packages/components/molecules/f-rating/stories/RatingMultiStar.stories.js]: Can we make all of these props kebab-cased, please? e.g.:

```suggestion
                    :review-count="rating.reviewCount"
```

File: packages/components/molecules/f-rating/src/components/_tests/Rating.test.js (hunk 1 above)
Comment (kevinrodrigues) [reply packages/components/molecules/f-rating/src/components/_tests/Rating.test.js]: I will need to set it to zero as it is a required prop.

**Review by xander-marjoram — DISMISSED** [review]

File: packages/components/molecules/f-rating/CHANGELOG.md [hunk 3]
```diff
@@ -3,6 +3,15 @@
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+v0.12.0
+------------------------------
+*November 21, 2022*
+
+### Added
+- `isUserRating` prop to control when (You) is displayed.
+- Tests included both unit and visual.
```
Comment (ashleynolan) [inline packages/components/molecules/f-rating/CHANGELOG.md]:
```suggestion
- Tests included – both unit and visual.
```

**Review by ashleynolan — COMMENTED** [review]
Tiny comment – otherwise LGTM

**Review by ashleynolan — DISMISSED** [review]

File: packages/components/molecules/f-rating/src/assets/scss/rating.scss [hunk 4]
```diff
@@ -35,10 +35,7 @@
...
 }
 
-.c-rating-star--filled path {
-    fill: f.$color-support-brand-01;
-}
-
+.c-rating-star--filled path,
 .c-rating-star--empty path {
-    fill: f.$color-mozzarella-50;
+    fill: f.$color-support-brand-01;
```
Comment (xander-marjoram) [inline packages/components/molecules/f-rating/src/assets/scss/rating.scss]: Is there a shared class that this rule could live in?

Comment (kevinrodrigues) [reply packages/components/molecules/f-rating/src/assets/scss/rating.scss:40]: Do you mean use a class within the vue component and then define here?

Comment (kevinrodrigues) [reply packages/components/molecules/f-rating/src/assets/scss/rating.scss]: Should be fixed now.

**Review by xander-marjoram — APPROVED** [review]

**Review by ashleynolan — APPROVED** [review]

**Pull request merged**